script.js

```diff
@@ -0,0 +1,621 @@
+  });
+  buttonsContainer.appendChild(exchangeOrHoldCardsButton);
+
+  // initialize buttonsContainer functionality
+  gameDisplayContainer.appendChild(buttonsContainer);
+
+ 
+
+  // initialize bidPointsInputEl functionality
+  bidPointsInputEl.setAttribute('type', 'text');
+  bidPointsInputContainer.appendChild(bidPointsInputEl);
+  // initialize bidPointsButton functionality
+  bidPointsButton.setAttribute('id', 'bid-points-button');
+  bidPointsButton.innerText = 'submit points';
+  bidPointsButton.addEventListener('click', () => {
+    if (canSubmitBidPoints === true) {
+      if (bidPointsInputEl.value > 0) { // player submitted a valid bid points
+        // player has submitted bid points so they cannot submit anymore in this round
+        canSubmitBidPoints = false;
+
+        // clear player's hand container display since a new round has started
+        playerHandContainer.innerHTML = '';
+
+        // store bid points and display it
+        playerBidPoints = bidPointsInputEl.value;
+        bidPointsInfoEl.innerText = playerBidPoints;
+        // clear bid points from bidPointsInputEl display
+        bidPointsInputEl.value = '';
+        // minus bid points away from player's total points
+        playerTotalPoints -= playerBidPoints;
+        // display player's total points
+        totalPointsInfoEl.innerText = playerTotalPoints;
+
+        // allow player to deal starting cards since it is a new round
+        canDealStartingCards = true;
+        // display instruction for player to click on dealButton to deal cards
+        displayGameInfo('draw');
+      } else {
+        // tell player to submit a valid bid points
+      }
+    } else {
+      // clear bid points from bidPointsInputEl display since
+      // player might have accidentally tried submitting
+      bidPointsInputEl.value = '';
+    }
+  });
+  bidPointsInputContainer.appendChild(bidPointsButton);
+  gameDisplayContainer.appendChild(bidPointsInputContainer);
+};
+
+// Initilize game
+initGame();
+
+ // Total points and bid points -------------------------------------------
+  // initialize bidPointsHeadingEl functionality
+  bidPointsHeadingEl.innerText = 'Points Bidded';
+  bidPointsContainer.appendChild(bidPointsHeadingEl);
+  // initialize bidPointsInfoEl functionality
+  bidPointsInfoEl.innerText = playerBidPoints;
+  bidPointsContainer.appendChild(bidPointsInfoEl);
+  // initialize bidPointsContainer functionality
+  gameDisplayContainer.appendChild(bidPointsContainer);
+  // initialize totalPointsHeadingEl functionality
+  totalPointsHeadingEl.innerText = 'Total Points';
+  totalPointsContainer.appendChild(totalPointsHeadingEl);
+  totalPointsContainer.appendChild(totalPointsHeadingEl);
+  // initialize totalPointsInfoEl  functionality
+  totalPointsInfoEl.innerText = playerTotalPoints;
+  totalPointsContainer.appendChild(totalPointsInfoEl);
+  // initialize totalPointsContainer functionality
+  gameDisplayContainer.appendChild(totalPointsContainer);
```

Comment:
consider putting this block of code into a function, and calling the function in `initGame`